dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/AbstractEventService.java

```diff
@@ -576,13 +576,15 @@ public Grid getEventsGrid( EventSearchParams params )
     public int getAnonymousEventValuesCountLastUpdatedAfter( Date lastSuccessTime )
     {
         EventSearchParams params = buildAnonymousEventsSearchParams( lastSuccessTime );
+        params.setIncludeDeleted( true );
         return eventStore.getEventCount( params, null );
```

Comment:
Should we rather have setIncludeDeleted( true) inside the buildAnonymousEventsSearchParams methods?